codespace-instructions.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+code .env
+echo \<------------ SETUP INSTRUCTIONS ------------\>
+echo 
+echo Almost done! Just need to set up a NextAuth Provider and run the app.
+echo
+echo 1.\) For dev environments, we have enabled use of the Github Provider. Please follow their guide:
+echo https://docs.github.com/en/developers/apps/building-oauth-apps/creating-an-oauth-app
+echo
+echo The mock user data URL is:   
+echo https://${CODESPACE_NAME}-3001.preview.app.github.dev/data
+echo
+echo The homepage is:   
+echo https://${CODESPACE_NAME}-3000.preview.app.github.dev
+echo 
+echo The callback url is:
+echo https://${CODESPACE_NAME}-3000.preview.app.github.dev/api/auth/callback/github
+echo
+echo 2.\) Set NEXTAUTH_URL in .env to https://${CODESPACE_NAME}-3000.preview.app.github.dev
+echo
+echo 3.\) To setup GITHUB_ID and GITHUB_SECRET after setting up the OAuth app: 
+echo      -Your Client ID is shown in the OAuth app page. Copy and paste to GITHUB_ID. 
+echo      -Next generate a new Client Secret. Then copy and paste to GITHUB_SECRET.
+echo
+echo Additional Steps to take After Finishing Gitpod Setup:
```

Comment:
```Suggestion
echo Additional Steps to take After Finishing Codespaces Setup:
```

Hi @utsab and @GuillermoFloresV 

There was a copy-and-paste minor typo:

Via https://github.com/freeCodeCamp/classroom/pull/349, I fixed it at https://github.com/freeCodeCamp/classroom/pull/349/files#diff-554f97261a97badb0ab3b9ed138c0408923adee822657cc18cd7ae86e5484b2bR25